examples/hosts/app-integration/external-controller/src/kvpair-dataobject/FluidStatic.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getObjectWithIdFromContainer } from "@fluidframework/aqueduct";
+import { Container } from "@fluidframework/container-loader";
+import { getTinyliciousContainer } from "@fluidframework/get-tinylicious-container";
+import { KeyValueContainerRuntimeFactory } from "./containerCode";
+
+export class FluidDocument {
+    constructor(private readonly container: Container, public readonly createNew: boolean) { }
+
+    public async createDataObject<T = any>(type: string, id: string) {
+        await this.container.request({ url: `/create/${type}/${id}` });
+        const dataObject = await this.getDataObject<T>(id);
+        return dataObject;
+    }
+
+    public async getDataObject<T = any>(id: string) {
+        const dataObject = await getObjectWithIdFromContainer<T>(id, this.container);
+        return dataObject;
+    }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class Fluid {
+    public static async createDocument(docId: string): Promise<FluidDocument> {
+        const container = await getTinyliciousContainer(docId, KeyValueContainerRuntimeFactory, true /* createNew */);
+        const document = new FluidDocument(container, true /* createNew */);
+        return document;
+    }
+    public static async getDocument(docId: string): Promise<FluidDocument> {
+        const container = await getTinyliciousContainer(docId, KeyValueContainerRuntimeFactory, false /* createNew */);
+        const document = new FluidDocument(container, false /* createNew */);
+        return document;
```

Comment:
I'm still interested in figuring out how we can improve these *and* where they should live. Probably related to #4610

Reply:
Makes sense - this PR shares current progress but agree this should be improved upon.  #4610 seems good to track with.